src/Microsoft.AspNet.Mvc.Razor/Precompilation/SymbolUtilities.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+#if DNXCORE50
+using System.Reflection;
+#endif
+using Microsoft.CodeAnalysis;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.Razor.Precompilation
+{
+    public static class SymbolUtilities
```

Comment:
Might need to change the name. We already have a `SymbolUtility` which deals with pdbs.